seed/java-sdk/imdb/disable-required-property-builder-checks/src/main/java/com/seed/api/resources/imdb/RawAsyncImdbClient.java

```diff
@@ -0,0 +1,159 @@
+/**
+ * This file was auto-generated by Fern from our API Definition.
+ */
+package com.seed.api.resources.imdb;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.seed.api.core.ClientOptions;
+import com.seed.api.core.MediaTypes;
+import com.seed.api.core.ObjectMappers;
+import com.seed.api.core.RequestOptions;
+import com.seed.api.core.SeedApiApiException;
+import com.seed.api.core.SeedApiException;
+import com.seed.api.core.SeedApiHttpResponse;
+import com.seed.api.resources.imdb.errors.MovieDoesNotExistError;
+import com.seed.api.resources.imdb.types.CreateMovieRequest;
+import com.seed.api.resources.imdb.types.Movie;
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.Headers;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import org.jetbrains.annotations.NotNull;
+
+public class RawAsyncImdbClient {
```

Comment:
Thoughts on `AsyncRawImdbClient`? I know why you named it like this to start, but I wonder if we should maintain that starting with `Async` means that we return a `CompleteableFuture` cc @dsinghvi